src/stores/auth.ts

```diff
@@ -19,6 +20,9 @@ export const useAuthStore = defineStore("auth", () => {
       user.value = netlifyIdentity.currentUser();
       return token;
     },
+    onSettled: () => {
```

Comment:
`useQuery` returns a ref `isLoading` so we shouldn't need to manage it on our own

Reply:
I cannot make use of isLoading() since that will return false before the query is run for the first time on fresh page load, triggering router/index.ts where this is used to stop waiting for login and check club access. I need to track when it has finished successfully loading for the first time and this is the best way I found to do that in my testing but let me know if theres a better way

Reply:
nvm found a better way lol, useQuery returns a ref `isInitialLoading` which is exactly what i was looking for